lib/user-interface/react/src/shared/modal/confirmation-modal.tsx

```diff
@@ -18,14 +18,13 @@ import { Modal as CloudscapeModal, Box, SpaceBetween, Button } from '@cloudscape
 import React, { ReactElement, useState } from 'react';
 import { useAppDispatch } from '../../config/store';
 import { dismissModal } from '../reducers/modal.reducer';
-import { MutationActionCreatorResult } from '@reduxjs/toolkit/query';
 
 export type CallbackFunction<T = any, R = void> = (props?: T) => R;
 
 export type ConfirmationModalProps = {
     action: string;
     resourceName: string;
-    onConfirm: () =>  MutationActionCreatorResult<any>;
+    onConfirm: () =>  any;
```

Comment:
Are we doing anything with the result? Should it be `void` instead of `any`?

Reply:
Trying to keep it future safe incase anything ever does anything with the response, but I could rip it out with our current use cases I suppose.